oceandb.ini

```diff
@@ -54,9 +54,9 @@ keeper.port = 8545
 keeper.network = development
 
 ;contracts.folder=venv/contracts
-market.address = 
-auth.address = 
-token.address = 
+market.address = 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66

```

Comment:
Can you please get rid of these specific addresses?

Reply:
next time I will follow squid example and create a oceandb.ini.local file and have it ignored in git, so this should not happen again